components/classes/content-experience.schema.json

```diff
@@ -0,0 +1,193 @@
+        }
+      }
+    },
+    "content-experience-featurization": {
+      "properties": {
+        "xdm:contentFeaturization": {
+          "title": "Content Featurization",
+          "description": "Content featurization.",
+          "type": "object",
+          "properties": {
+            "xdm:keywords": {
+              "title": "Experience Keywords",
+              "type": "array",
+              "description": "Experience keywords.",
+              "items": {
+                "type": "string"
+              }
+            },
+            "xdm:namedEntities": {
+              "title": "Experience Named Entities",
+              "type": "array",
+              "description": "Experience named entities.",
+              "items": {
+                "type": "string"
+              }
+            },
+            "xdm:persuasionStrategies": {
+              "title": "Experience Persuasion Strategies",
+              "type": "array",
+              "description": "Experience persuasion strategies.",
+              "items": {
+                "type": "string"
+              }
+            },
+            "xdm:emotions": {
+              "title": "Experience Emotions",
+              "type": "array",
+              "description": "Experience emotions.",
+              "items": {
+                "type": "string"
+              }
+            },
+            "xdm:visualContentDensity": {
+              "title": "Experience Visual Content Density",
+              "type": "string",
+              "description": "Experience visual content density such as low, medium or high."
+            },
+            "xdm:visualAttentionSpread": {
+              "title": "Experience Visual Attention Spread",
+              "type": "string",
+              "description": "Experience visual attention spread such as low, medium or high."
+            },
+            "xdm:wordsCount": {
+              "title": "Experience Words Count",
+              "type": "integer",
+              "description": "Experience words Count."
+            },
+            "xdm:sentencesCount": {
+              "title": "Experience Sentences Count",
+              "type": "integer",
+              "description": "Experience Sentences Count."
+            },
+            "xdm:hashTagsCount": {
+              "title": "Experience Hash Tags Count",
+              "type": "integer",
+              "description": "Experience hash tags count."
+            },
+            "xdm:emojisCount": {
+              "title": "Experience Emojis Count",
+              "type": "integer",
+              "description": "Experience emojis count."
+            },
+            "xdm:textQuotesCount": {
+              "title": "Experience Text Quotes Count",
+              "type": "integer",
+              "description": "Experience text quotes count."
+            },
+            "xdm:wordsCountPerSentence": {
+              "title": "Experience Words Count Per Sentence",
+              "type": "number",
```

Comment:
Why is this field not an integer?

Reply:
this will be the value of count(words)/count(sentences) from the experience.
Won't be round to an integer in any way tho, so we will expect float values.
Similar to stopWordsRation.

Reply:
Ok. Do you want to add a "minimum": 0 constraint to prevent a negative value? You would probably want to add such a constraint to all "count" fields